main.js

```diff
@@ -9,51 +9,130 @@ const RARITY = {
     LEGENDARY: 0.5,
 }
 
-const Item = require('./server/schemas/item');
-const Ornament = require('./server/schemas/ornament');
-const Settings = require('./server/schemas/settings');
+const { Item } = require('./server/schemas/item');
+const { Ornament } = require('./server/schemas/ornament');
+const { Player }= require('./server/schemas/player');
+const { Creature }= require('./server/schemas/creature');
+
+const { createItem } = require('./server/services/items-service')
+
 const { token, databaseToken } = process.env; 
 const { connect, connection } = require('mongoose');
+const { Community } = require("./server/schemas/community");
+
+const { createItem } = require("./createHelper")
 
 async function main() {
 
     console.log("The elves are getting to work...")
     await connect(databaseToken, {dbName: 'christmas_bot_v2'}).catch(console.error);
     console.log("Christmas Bot Status: ONLINE");
 
-    // await createItem('Gumdrop Button', RARITY.COMMON);
-    // await createOrnament('Gumdrop Ornament', RARITY.LEGENDARY, undefined, 123);
-    // await initializeSettings(123);
+
+    //TESTING: set to true if you want to purge databases
+    let input = true
+    if (input) {
+        console.log("Deleting existing databases...");
+        await connection.db.dropDatabase();
+    }
+
+    const myItem = await createItem('Gumdrop Button', RARITY.COMMON, `google.com`);
+
+    const myOrnament = await createOrnament('Gumdrop Ornament', RARITY.LEGENDARY, "", 123);
+
+    const myCreature = await createCreature("Grim Grimmler", "he", [myItem, myOrnament], "naughty", "" );
+    const myPlayer = await createPlayer(123, 456, 1, [myItem], [myOrnament], 0);
+    
+    await console.log("Attemping to create community...")
+    const myCommunity = await createCommunity(456, [myOrnament], [myCreature]);
+
+
 
     await connection.close();
     console.log("Christmas Bot Status: OFFLINE");
 }
 
-async function createItem(name, rarity, image) {
-    const item = new Item({name, rarity, image})
+async function createOrnament(name, rarity, image, serverId, isFound ) {
```

Comment:
I assume the plan is to eventually move this to a service file like item-service?